docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -2131,6 +2132,7 @@ to_unixtime(expression[, ..., format_n])
 - [empty](#empty)
 - [flatten](#flatten)
 - [generate_series](#generate_series)
+- [list_any_value] (#list_any_value)
```

Comment:
I think the space breaks the markdown rendering. Opened https://github.com/apache/datafusion/pull/12627
<img width="1584" alt="image" src="https://github.com/user-attachments/assets/3cb77641-ce4c-45a3-a66f-edc6ed5e1e3a">